src/meta/expressionEvaluator/functions/validatorColSumNotGreaterThanForest.ts

```diff
@@ -0,0 +1,28 @@
+import { ExpressionFunction } from '@openforis/arena-core/dist/expression/function'
+import { Numbers } from '@utils/numbers'
+import { Objects } from '@utils/objects'
+
+import { NodeValueValidation, NodeValueValidationMessage } from '@meta/assessment'
+
+import { Context } from '../context'
+
+export const validatorColSumNotGreaterThanForest: ExpressionFunction<Context> = {
```

Comment:
isn't this validatorNotGreaterThanForest?

Reply:
or maybe validatorNotGreaterThanMaxForest?

Reply:
This was used for 5a and it shows an error message "The sum of <year> ..."

```
"validatorColSumNotGreaterThanForest((disturbances.insects || 0) + (disturbances.diseases || 0) + (disturbances.severe_weather_events || 0) + (disturbances.other || 0), (extentOfForest.forestArea || maxForestArea()) )"
```

Should it still be renamed?

Reply:
![image](https://user-images.githubusercontent.com/5508251/232706078-b64193e1-cceb-4114-9837-0a9e934b5586.png)